History.txt

```diff
@@ -1,11 +1,18 @@
 Documentation and download available at https://www.FreeRTOS.org/
 
-Changes after FreeRTOS 202012.00 release
+Changes after FreeRTOS 202012.00-LTS and 202012.01-LTS releases
 
+		+ Update AWS IoT Device Defender and AWS IoT Jobs libraries
+		  to LTS releases.
+		+ Added new OTA library.

```

Comment:
Need to reword to make sure this new OTA library is v3.0.0, same as the LTS version released in FreeRTOS-LTS 202012.01. Same to Device Defender and Jobs.

Reply:
I have explicitly mentioned the LTS release versions of the libraries